apps/jsonrpc2/test/jsonrpc2/response/block_test.exs

```diff
@@ -2,11 +2,12 @@ defmodule JSONRPC2.Response.BlockTest do
   use ExUnit.Case, async: true
 
   alias JSONRPC2.Response.Block
-  alias JSONRPC2.TestFactory
+
+  import JSONRPC2.TestFactory
```

Comment:
Does `import` give us any benefits over `alias`? If not, it's a bit clearer to `alias` because then someone unfamiliar with the code knows where `build` is defined.